deploy/skaffold/distribution.dockerfile

```diff
@@ -0,0 +1,114 @@
+    chmod +x kubectl && \
+    mv kubectl /usr/local/bin/ && \
+    rm -rf /tmp/*
+
+# Download helm
+ENV HELM_VERSION v2.12.0
+ENV HELM_URL https://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-linux-amd64.tar.gz
+RUN curl -sSfLo helm.tar.gz "${HELM_URL}" && \
+    tar -xvf helm.tar.gz --strip-components 1 && \
+    mv helm /usr/local/bin/ && \
+    rm -rf /tmp/*
+
+# Download kustomize
+ENV KUSTOMIZE_VERSION 2.1.0
+ENV KUSTOMIZE_URL https://github.com/kubernetes-sigs/kustomize/releases/download/v${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_amd64
+RUN curl -sSfLo kustomize "${KUSTOMIZE_URL}" && \
+    chmod +x kustomize && \
+    mv kustomize /usr/local/bin/ && \
+    rm -rf /tmp/*
+
+# Download kompose
+ENV KOMPOSE_VERSION v1.18.0
+ENV KOMPOSE_URL https://github.com/kubernetes/kompose/releases/download/${KOMPOSE_VERSION}/kompose-linux-amd64
+RUN curl -sSfLo kompose "${KOMPOSE_URL}" && \
+    chmod +x kompose && \
+    mv kompose /usr/local/bin/ && \
+    rm -rf /tmp/*
+
+# Download container-structure-test
+ENV CONTAINER_STRUCTURE_TEST_VERSION v1.5.0
+ENV CONTAINER_STRUCTURE_TEST_URL https://storage.googleapis.com/container-structure-test/${CONTAINER_STRUCTURE_TEST_VERSION}/container-structure-test-linux-amd64
+RUN curl -sSfLo container-structure-test "${CONTAINER_STRUCTURE_TEST_URL}" && \
+    chmod +x container-structure-test && \
+    mv container-structure-test /usr/local/bin/ && \
+    rm -rf /tmp/*
+
+# Download kind
+ENV KIND_VERSION v0.6.0
+ENV KIND_URL https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-linux-amd64
+RUN curl -sSfLo kind "${KIND_URL}" && \
+    chmod +x kind && \
+    mv kind /usr/local/bin/ && \
+    rm -rf /tmp/*
+
+# Download bazel
+ENV BAZEL_VERSION 0.27.0
+ENV BAZEL_URL https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-linux-x86_64
+RUN curl -sSfLo bazel "${BAZEL_URL}" && \
+    chmod +x bazel && \
+    mv bazel /usr/local/bin/ && \
+    rm -rf /tmp/*
+RUN bazel version
+
+# Download pack
+ENV PACK_VERSION 0.4.1
+ENV PACK_URL https://github.com/buildpack/pack/releases/download/v${PACK_VERSION}/pack-v${PACK_VERSION}-linux.tgz
+RUN curl -sSfLo pack.tgz "${PACK_URL}" && \
+    tar -zxf pack.tgz && \
+    mv pack /usr/local/bin/ && \
+    rm -rf /tmp/*
+
+# Download gcloud
+ENV GCLOUD_VERSION 245.0.0
+ENV GCLOUD_URL https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${GCLOUD_VERSION}-linux-x86_64.tar.gz
+RUN curl -sSfLo gcloud.tar.gz "${GCLOUD_URL}" && \
+    tar -zxf gcloud.tar.gz && \
+    mv google-cloud-sdk / && \
+    rm -rf /tmp/*
+RUN CLOUDSDK_PYTHON="python2.7" /google-cloud-sdk/install.sh \
+    --usage-reporting=false \
+    --bash-completion=false \
+    --disable-installation-options
+ENV PATH=$PATH:/google-cloud-sdk/bin
+RUN gcloud auth configure-docker
+
+# Distribution specific
+COPY --from=build-skaffold /skaffold/out/skaffold-linux-amd64 /usr/bin/skaffold
```

Comment:
maybe i'm missing something here, but won't this only build the linux binary? this dockerfile is being used by `make cross` so i'm not sure how it will work.